### PR TITLE
It's 2020, time to upgrade this text.

### PR DESCRIPTION
**Pull request by bartek**

### CODE REVIEW

**Comment by marktani** [comment]
Thanks this is a great change! I have never encountered this string in-game 🤔

**Comment by bartek** [comment]
@marktani Heh, yeah. It only shows up if you pick `Blain` as a bot to play against.

**Comment by furai** [comment]
https://youtu.be/HayFxnfFgZk << This is the original Blain's line from the Predator film. That's how all the bots were done. They're characters from the various films. Just giving more context.

**Comment by bartek** [comment]
@furai I get the need to respect an original source but as a society we have to adapt to a language that is more commonly accepted. 

Are you the maintainer of this repository? I think so, but don't want to assume.

**Comment by gitMM** [comment]
It is a quote from the Predator movie but my guess is many people don't know this and it might be interpreted wrongly.

**Comment by marktani** [comment]
If we don't want to "spoil" an original source we could also remove this line entirely as an alternative.

**Comment by gitMM** [comment]
Or replace the words with ****** to preserve the fact it's a quote.

**Comment by bartek** [comment]
@gitMM I am happy with this and can make the amendment, assuming the owner (who's that?) is good with it!

**Comment by Akinaro** [comment]
Question.... No one was bother with that text in game where you literally blow people to peaces... So why now you show up and ask to change QUOTE from amazing movie? 
What is wrong with it?

**Comment by furai** [comment]
@bartek can you update your PR to change the quote to asterisks like MM mentioned? Thanks.
Alternatively we could find another quote from Blain to replace it with.

**Comment by bartek** [comment]
@furai I took a look at other Blain quotes but none that I can identify from what IMBD provides that fit the _theme_ of this one, so I opted for the asterisk. 

I think @marktani's line of thinking is the most appropriate. There's a large debate we could have with regards to preserving history vs. updating for modern times, but that'd need to be a holistic conversation which considers the source of all the quotes within Soldat (which seem to be exclusively from Predator)

I don't want to blow this out of scope though and I appreciate making this incremental step forward.

**Comment by You-Got-Served** [comment]
It would be fantastic if my objections to this pull request were taken into consideration. 

I would like to state that offense is subjective. Whilst one may be offended by a particular word, another may take equal offense to this word being censored. Or a different word entirely...

If you tread this path, where does it end? Removing “I am the law!” in solidarity with US citizens plight of an oppressive police force? Removing “What the hell are you?” to appease religious sentiments? These are legitimate claims if you set this precedent of censorship.

It is important to remember the concept behind Soldat is 80s/90s action movies. They are crude & comedic. Soldat has always had a mature content rating.

If I remember correctly, Bartek raised this issue previously on Soldat Forums - where it gained no support. It is a little disappointing to now see it get approved, & without consulting the Soldiers. (The people who keep this game alive.)

It is maybe relevant to add that there is a big movement within the LGBT community to reclaim the f word in question. You will commonly hear this word used by LBGT individuals.

However, you are in luck, Bartek. There are multiple ways you can resolve this personal issue already implemented into the game! :D

It is possible to disable Bot chat in the Soldat options tab. You can disable Bot chat on a Soldat server very simply by editing the soldat.ini text file.
You can open the Blain bot file and edit out that taunt in your client.
You can /mute Blain if it ever becomes a problem for you in an online game. This all really is very accommodating to your needs and I am happy to provide you with the resolution you were seeking.

Also, Akinaro has given an abundance of his time to the Soldat community in the form of content creation & community interaction. Please respect him enough to allow him his voice.

Many thanks

Tom

**Comment by bartek** [comment]
@You-Got-Served All valuable commentary!

> I would like to state that offense is subjective. Whilst one may be offended by a particular word, another may take equal offense to this word being censored. Or a different word entirely...

> If you tread this path, where does it end? Removing “I am the law!” in solidarity with US citizens plight of an oppressive police force? Removing “What the hell are you?” to appease religious sentiments? These are legitimate claims if you set this precedent of censorship.

Indeed, this can go far. Making iterative progress is important even when it's imperfect. If we asked the questions you're asking, it'd require a rethink as to where Soldat should acquire its style and substance from. Today, it's from Predator, and I think that's fine, but that doesn't negate us from making incremental progress towards a more inclusive game.

> It is maybe relevant to add that there is a big movement within the LGBT community to reclaim the f word in question. You will commonly hear this word used by LBGT individuals.

The reality is that faggot is today seen as a pejorative term, and has been that way for a long time. Yes, there are [some](https://www.gq-magazine.co.uk/lifestyle/article/faggot-meaning) [discussions](https://www.advocate.com/arts-entertainment/2017/8/02/21-words-queer-community-has-reclaimed-and-some-we-havent#media-gallery-media-6) about reclaiming it, but that is a reclamation of the word *by the LGBTQ* community, not any community. 

> If I remember correctly, Bartek raised this issue previously on Soldat Forums - where it gained no support. It is a little disappointing to now see it get approved, & without consulting the Soldiers. (The people who keep this game alive.)

Ah, you must have me confused by some other Bartek. Mine's a very common name in Poland and as expected, only when I am in Poland or in Polish communities do I get confused with others. My history with Soldat is playing it when I was in college and I came back here when I realized it was open source.

I spun the game up with a friend of mine (we're now in our 30s) and we had a great time, but winced when we saw Blain's message.

> It is possible to disable Bot chat in the Soldat options tab. You can disable Bot chat on a Soldat server very simply by editing the soldat.ini text file.

These are all good options but it's like telling a person of the LGBTQ community to cover their eyes when they enter your home, just in case they can't handle the things on your wall. Why not make your home more inclusive?

Soldat already bleeps out other words, but I imagine there wasn't a ton of argument against them. [Sgt. Mac](https://github.com/Soldat/base/blob/master/server/bots/Sgt.%20Mac.bot), [Dutch](https://github.com/Soldat/base/blob/master/server/bots/Dutch.bot), and [Commando](https://github.com/Soldat/base/blob/master/server/bots/Commando.bot) all have phrases bleeped that objectively, reference words which don't put down entire communities (e.g. `fuck`)

This has obviously grown bigger than I anticipated but ultimately, I am glad we're *having a discussion* about it, so thank you Tom & everyone else involved thus far.

**Comment by Deide** [comment]
If you take incremental "progress" to its logical conclusion, it means removing the game altogether. Let me explain.

If inclusivity is your ultimate goal, then there's also no argument against lowering the overall skill ceiling. There's also no argument against removing depictions of violence - there are a minority of people that are against it after all. In the end, matches would be decided by pure RNG, and those that got randomly lucky would be penalised so that everyone is equal and included.

It's not that I even disagree with the noble sentiment of wanting to make the community a happier place, but the originally proposed change not only compromises the artistic vision (out of sheer misunderstanding or ignorance, so you'd be making the people that are happy with the 80s/90s action hero aesthetic - the same people that have played the game for more than a decade - less happy), but is also potentially offensive (_not_ discriminatory) to people brought up in rural areas (of which there are many).

In the end, a compromise is the asterisk censorship as mentioned, but that also isn't ideal to me personally. If you want to market the game to children, it's understandable, but most of us are adults that don't really need to be told what we can and can't handle, especially when the asterisks don't suddenly destroy the word (ie. you still know what word it is).

The _best_ compromise is to have a box you can tick in settings to censor the game for yourself. Or some multi-state widget/slider that lets people completely remove material. Luckily there are already ways to do this but they're not exactly clear when you open the game, so it should instead come down to an issue of clarity and accessibility or defaults.

TLDR: Inclusivity and Progress are not necessarily synonymous with positive change; we should respect artistic vision and individual choice.

**Comment by bartek** [comment]
Sorry @Deide but I find your response very frustrating and you won't get much time from me.

> If you take incremental "progress" to its logical conclusion, it means removing the game altogether. Let me explain.

I've seen this argument time and time again from scared conservatives who worry people not like them are going to _take over_ their neighbourhoods/schools/children's minds.

It's a stretch and one that has absolutely no substance.

> It's not that I even disagree with the noble sentiment of wanting to make the community a happier place, but the originally proposed change not only compromises the artistic vision (out of sheer misunderstanding or ignorance, so you'd be making the people that are happy with the 80s/90s action hero aesthetic - the same people that have played the game for more than a decade - less happy), but is also potentially offensive (not discriminatory) to people brought up in rural areas (of which there are many).

Mhm. The original adjustment was based off ignorance but I have since been educated on what the term means, so thank you! 

> In the end, a compromise is the asterisk censorship as mentioned, but that also isn't ideal to me personally.

This isn't about you. It's about those around you

> If you want to market the game to children, it's understandable,

I never claimed this and it's not the goal. We have rating systems in place for this anyways.

>  but most of us are adults that don't really need to be told what we can and can't handle,

Have you ever taken a second to be in someone else's shoes? Specifically someone from an underrepresented community?

> especially when the asterisks don't suddenly destroy the word (ie. you still know what word it is).

Perfect, then let's remove the line all together!

So, here's my proposal as it stands:

1. To make it clearer what the Soldat community stands for and against, it should adopt a [Code Of Conduct](https://opensource.guide/code-of-conduct/). I am happy to help provision this but I also have limited history with the Soldat community.

2. Assuming the above aligns with the motivations of this PR, remove the line all together. This does not compromise the _artistic vision_, nor put _words into peoples heads_, so I'd consider it a win-win.

**Comment by BranDougherty** [comment]
Better than nothing, and it has the MM stamp of approval. I don't plan on implementing anyone's particular ideology into the game, I'd just like to avoid slapping people in the face with slurs before a serious release. Let's go with this.

Done in d9aaef527a0efa46184d65470283594261e86bbe.
